src/www/css/app.css

```diff
@@ -768,10 +788,6 @@ video {
   height: 3rem;
 }
 
-.h-14 {
```

Comment:
Can I just check that these are expected deletions?

Reply:
Yep this changes where done by @suxscribe and not by CI see:
https://github.com/wg-easy/wg-easy/commit/7d19b748452bbf9832974704219bf48a5d10f521